pandas/tests/groupby/test_reductions.py

```diff
@@ -1047,6 +1047,120 @@ def scipy_sem(*args, **kwargs):
     return sem(*args, ddof=1, **kwargs)
 
 
+@pytest.mark.parametrize(
+    "reduction_method",
+    ["sum", "min", "max", "mean", "median", "prod", "sem", "std", "var"],
+)
+def test_skipna_reduction_ops_cython(reduction_method):
+    # GH15675
+    # Testing the skipna parameter against possible datatypes
+    df = DataFrame(
+        {
+            "l": ["A", "A", "A", "A", "B", "B", "B", "B"],
+            "int": [-1, 1, -1, 2, 1, 1, 1, np.nan],
+            "float": [-1.0, 1.2, -1.1, 1.5, -1.1, 1.5, np.nan, 1.0],
```

Comment:
Edited it, although I'm not entirely sure if we made the right parametrization. We will apply any further changes needed.

Reply:
Sorry - I should have been more clear. Each assert only uses a single column. So I think you can do:

```python
@pytest.mark.parametrize(
    "reduction_method, values",
    [
        ("sum", [-1.0, 1.2, -1.1, 1.5, -1.1, 1.5, np.nan, 1.0]),
        ("sum", ["foo", "bar", "baz", "foo", "foo", "foo", pd.NA, "foo"]),
    ],
)
def test_skipna_reduction_ops_cython(reduction_method, values):
    df = pd.DataFrame({"key": [1, 1, ...], "values": values})
    gb = df.groupby("key")
    result = getattr(gb, reduction_method)(skipna=False)
    ...
```
This makes it easy to expand on if necessary, and only one test per parametrization (rather than looping over columns).

Reply:
Understood, this does make the parametrization take quite a few lines but I agree that it's easier to expand and seems more consistent with that's done in other tests.